CHANGELOG.md

```diff
@@ -257,6 +257,9 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/).
   - **CUMULUS-2468** - Added `@cumulus/aws-client/DynamoDb.parallelScan` helper to perform [parallel scanning on DynamoDb tables](https://docs.aws.amazon.com/amazondynamodb/latest/developerguide/Scan.html#Scan.ParallelScan)
   - **CUMULUS-2507**
     - Updated granule record write logic to set granule status to `failed` in both Postgres and DynamoDB if any/all of its files fail to write to the database.
+- **CUMULUS-2371**
```

Comment:
These changelog notes need to be moved under `Unreleased`. Currently this is under release 9.0.0

Reply:
[`5223038` (#2132)](https://github.com/nasa/cumulus/pull/2132/commits/522303829adf7b5c63fec589dced18ebcf084cf4)